tools/cdi-func-test-proxy/main.go

```diff
@@ -10,30 +10,39 @@ import (
 	"log"
 	"net"
 	"net/http"
-	"os"
+	"path/filepath"
 	"strings"
 	"sync"
 	"time"
-)
 
-type key int
+	"k8s.io/klog/v2"
 
-const (
-	contentTypeKey   = "Content-Type"
-	contentTypeText  = "text/plain"
-	proxyHeaderKey   = "X-GoProxy"
-	proxyHeaderValue = "import-proxy-test"
-	username         = "foo"
-	password         = "bar"
-	httpPort         = "8080"
-	httpPortWithAuth = "8081"
-	withBasicAuth    = true
-	noBasicAuth      = false
+	"github.com/pkg/errors"
+	"kubevirt.io/containerized-data-importer/pkg/util"
+	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
-var logger *log.Logger
+const (
+	contentTypeKey    = "Content-Type"
+	contentTypeText   = "text/plain"
+	proxyHeaderKey    = "X-GoProxy"
+	proxyHeaderValue  = "import-proxy-test"
+	username          = "foo"
+	password          = "bar"
+	httpPort          = "8080"
+	httpPortWithAuth  = "8081"
+	httpsPort         = "443"
+	httpsPortWithAuth = "444"
+	withBasicAuth     = true
+	noBasicAuth       = false
+	serviceName       = "cdi-test-proxy"
+	configMapName     = serviceName + "-certs"
+	certFile          = "tls.crt"
+	keyFile           = "tls.key"
+	certDir           = "/certs"
+)
 
-func startServer(port string, basicAuth bool, wg *sync.WaitGroup) {
+func startServer(port string, basicAuth bool, useTls bool, wg *sync.WaitGroup) {
```

Comment:
linter complains `tools/cdi-func-test-proxy/main.go:45:47: func parameter useTls should be useTLS`

Reply:
Fixed